composer.lock

```diff
@@ -13933,6 +13930,7 @@
                 "mock",
                 "xunit"
             ],
+            "abandoned": true,
```

Comment:
This is worrying but out of scope for this PR. I made a followup for this: https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-5112